gwctl/pkg/printer/httproutes.go

```diff
@@ -48,7 +51,17 @@ func (hp *HTTPRoutesPrinter) Print(resourceModel *resourcediscovery.ResourceMode
 			hostNamesOutput = fmt.Sprintf("%v + %v more", strings.Join(hostNames[:2], ","), cnt-2)
 		}
 
-		row := []string{httpRouteNode.HTTPRoute.Name, hostNamesOutput}
+		parentRefsCount := fmt.Sprintf("%v", len(httpRouteNode.HTTPRoute.Spec.ParentRefs))
```

Comment:
- `gwctl get httproute -o wide`
```
NAMESPACE  NAME                 HOSTNAMES                          PARENT REFS               AGE  POLICIES
default    foo-httproute-1      example.com,example2.com + 1 more  ns2/demo-gateway-2        5m   2
default    qmn-httproute-100    example.com                        demo-gateway-1            5m   1
ns1        bar-route-21         foo.com,bar.com + 5 more           default/demo-gateway-200  5m   3
ns2        bax-httproute-18777  None                               ns1/demo-gateway-345      5m   4
```

In the [Sample outputs](https://deploy-preview-2723--kubernetes-sigs-gateway-api.netlify.app/geps/gep-2722/#sample-outputs), `PARENT REFS` outputs namespaces and name, but the [description](https://deploy-preview-2723--kubernetes-sigs-gateway-api.netlify.app/geps/gep-2722/#output-formats) explains that it outputs count of parent references.